projects/components/src/datagrid/interfaces/datagrid-action-display.interface.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+import {
+    ActionStyling,
+    ContextualActionDropdownDisplayConfig,
+    ContextualActionInlineDisplayConfig,
+    TextIcon,
+} from '../../common/interfaces';
+import { ContextualActionPosition } from '../datagrid.component';
+
+/**
+ * Makes the featured count as required when action menu is displayed as a dropdown and also adds the position
+ * at which the actions have to displayed in a datagrid
+ */
+interface DatagridContextualActionDropdownDisplayConfig extends ContextualActionDropdownDisplayConfig {
+    /**
+     * An enum that describes where the contextual buttons should display.
+     */
+    position: ContextualActionPosition;
```

Comment:
Do you think it will be better to rename `ContextualActionPosition` to `DatagridContextualActionPosition`, because  when I read `ContextualActionPosition` I'm thinking contextual action position in general